survey_contact_generation/views/survey_question_views.xml

```diff
@@ -10,6 +10,23 @@
                     <field name="allowed_field_ids" invisible="1" />
                 </group>
             </xpath>
+            <xpath expr="//field[@name='matrix_row_ids']" position="attributes">
```

Comment:
This isn't the right new field. Moreover, you're squashing the previous context with a value that doesn't correspond to it.
```suggestion
            <xpath expr="//field[@name='suggested_answer_ids']" position="attributes">
```